### ISSUE TITLE
Improve UI look & feel

### ISSUE THREAD

**Issue by lidel** [ipfs/public-gateway-checker#79]
Some time ago @fsdiogo made this:

> ![a](https://user-images.githubusercontent.com/157609/70870531-c4776000-1f94-11ea-9161-05f4d94f0f93.png)

Creating PR with a port of above design could be a good "first issue"
Sources are at: https://github.com/fsdiogo/ipfs-infra-status

**Comment by lazyweirdo** [comment]
yep, i want to work on this, and many other things, but it takes astonishingly long to somebody with write access to answer a simple PR ... truly demotivating

**Comment by lidel** [comment]
I think this repo slipped through fingers and nobody was responsible for issue/pr triage and maintenance. It should improve after holidays, when more people get write access.

**Comment by jamiew** [comment]
I was the standing maintainer but tagged Lidel and others this morning about getting some more eyeballs on this project. Thanks for all your patches so far

**Comment by lazyweirdo** [comment]
@jamiew you seem to be the only one active with write access ... merge man ... i love you ... i am trying to help ... i know, patience is not my business, im sure you guessed that ... it is a small project, let's speed it up a little bit, help me ... kisses :kissing_heart::kissing_heart::kissing_heart:

**Comment by jamiew** [comment]
This is just an issue, not a PR. PR welcome

**Comment by jamiew** [comment]
Looks like this is all done if we did want to make this React-based: https://github.com/fsdiogo/ipfs-infra-status

This is also technically a duplicate of #60

**Comment by lazyweirdo** [comment]
> This is just an issue, not a PR. PR welcome

@jamiew ... i was asking for merging, in general, not specifically this 'issue' ... then i saw you merged #78  and not #76  which is more important since it fixes some bugs ... making #78 fall in conflict

i give up ... i love you all, anyway ... :kissing:

**Comment by jamiew** [comment]
@lazyweirdo per thread in #76 there are outstanding changes requested. 

It's also hard to follow a conversation across multiple threads so any future notes on that one welcome over there instead. Thanks!

**Comment by lazyweirdo** [comment]
it is not that hard man ... come on ... :kissing_heart: ... i just feel a little bit disenchanted

**Comment by jamiew** [comment]
Closed by #83
